### PR TITLE
[3006.x] Treat version as json string not number

### PR DESCRIPTION
**Pull request by dwoz**
Treat version as json string.

### CODE REVIEW

**Comment by babs** [comment]
I guess I'll have to wait for this one to finish #59927 (and rollback my last commit then) ;)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -223,7 +223,7 @@ jobs:
 
       - name: Check Salt Releases
         run: |
-          echo '${{ steps.get-salt-releases.outputs.latest-release }}' | jq -C '.'
+          echo \"${{ steps.get-salt-releases.outputs.latest-release }}\" | jq -C '.'
```
Comment (s0undt3ch) [inline .github/workflows/ci.yml:226]:
```suggestion
          echo '"${{ steps.get-salt-releases.outputs.latest-release }}"' | jq -C '.'
```
And we avoid escapes?

**Review by s0undt3ch — DISMISSED** [review]

**Comment by s0undt3ch** [comment]
FYI, push a commit.

**Comment by s0undt3ch** [comment]
The right fix, https://github.com/saltstack/salt/pull/65821

**Pull request closed without merging**